sympy/vector/coordsysrect.py

```diff
@@ -98,6 +98,33 @@ def __new__(cls, name, location=None, rotation_matrix=None,
             location = Vector.zero
             origin = Point(name + '.origin')
 
```

Comment:
We do it the general way, no need to complicate the code with special cases. Translations are subsets of transformations. In the special cases of rotation and translations, we could store some markers to remind of the special case (and remember the steps).